prime-angular-frontend/Dockerfile

```diff
@@ -19,37 +19,36 @@ ENV KEYCLOAK_CLIENT_ID $KEYCLOAK_CLIENT_ID
 ENV JWT_WELL_KNOWN_CONFIG $JWT_WELL_KNOWN_CONFIG
 RUN apt-get update && \
     apt-get install -y nginx gettext-base && \
-    mkdir -p /var/cache/nginx && \ 
-    mkdir -p /var/cache/nginx/client_temp && \ 
-    touch /etc/nginx/conf.d/default.conf && \ 
-    chmod -R 777 /etc/nginx && \ 
-    chmod -R 777 /var/cache/nginx && \ 
+    mkdir -p /var/cache/nginx && \
+    mkdir -p /var/cache/nginx/client_temp && \
+    touch /etc/nginx/conf.d/default.conf && \
+    chmod -R 777 /etc/nginx && \
+    chmod -R 777 /var/cache/nginx && \
     chmod -R 777 /var/run
 COPY nginx.conf /etc/nginx/
 COPY nginx.template.conf /etc/nginx/nginx.template.conf
 COPY entrypoint.sh /
 RUN echo "Populating environment..." && \
     (eval "echo \"$(cat /usr/src/app/src/environments/environment.prod.template.ts )\"" ) > /usr/src/app/src/environments/environment.prod.ts
 RUN cat /usr/src/app/src/environments/environment.prod.ts
-RUN npm install @angular/cli -g && \ 
-    npm install && \ 
-    npm i @angular-devkit/build-angular@0.803.24 && \ 
-    ng build --prod && \ 
+RUN npm install @angular/cli -g && \
+    npm install && \
+    npm audit fix && \
+    ng build --prod && \
     echo "NPM packages installed..."
 
-# RUN npm audit fix --only=prod && \ 
-#     npm install @angular-devkit/build-angular@0.803.24 \ 
+# RUN npm audit fix && \
```

Comment:
Not sure if this is needed commented out here as we are including it above.

Reply:
Yah, probably not, but it's Paul's code and I don't know if he had something in mind when he commented it out.  @poggdogg do we need this commented out code in the Dockerfile?

Reply:
Was commented out during troubleshooting of angular 8 - please delete.